pages/index.js

```diff
@@ -1,3 +1,25 @@
+import Link from 'next/link';
+import { React, useState } from 'react';
+import styled from 'styled-components';
+
 export default function Home() {
-  return <div></div>;
+  return (
+    <Container>
+      <Link href="/rooms">
+        <Button>Show all rooms</Button>
+      </Link>
+    </Container>
+  );
 }
+
+const Button = styled.button`
```

Comment:
This should be an anchor tag instead of a button. By using an anchor tag you ensure the accessibility of your links.

After changing the `button` tag to an `a` tag, you should also think about renaming the styled component.

Reply:
Done.